docs/source/markdown/podman-save.1.md

```diff
@@ -9,8 +9,8 @@ podman\-save - Save image(s) to an archive
 **podman image save** [*options*] *name*[:*tag*]
 
 ## DESCRIPTION
-**podman save** saves an image to either **docker-archive**, **oci-archive**, **oci-dir** (directory with oci manifest type), or **docker-dir** (directory with v2s2 manifest type) on the local machine,
-default is **docker-archive**. **podman save** writes to STDOUT by default and can be redirected to a
```

Comment:
I’m not sure we want to drop the mention of a default from the description completely. 

If we want to preserve it, I’d prefer something like
`The default output format is tar archive interoperable with **docker load(1)**` so that users don’t have to search the rest of the man page to figure out what the option value means.